#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_fix_path_import.dpatch by  <Serafeim Zanikolas <serzan@hellug.gr>>
##
## DP: fix path import

@DPATCH@
diff -ru ipython-0.8.4.orig/IPython/Extensions/ipipe.py ipython-0.8.4/IPython/Extensions/ipipe.py
--- ipython-0.8.4.orig/IPython/Extensions/ipipe.py	2008-05-31 23:31:42.000000000 +0100
+++ ipython-0.8.4/IPython/Extensions/ipipe.py	2008-08-19 21:02:53.000000000 +0100
@@ -124,7 +124,7 @@
 
 from IPython.external import simplegeneric
 
-import path
+from IPython.external import path
 
 try:
     from IPython import genutils, generics
